metadata-ingestion/examples/ai/dh_sdk_client_sample.py

```diff
@@ -0,0 +1,123 @@
+import argparse
+from datetime import datetime
+
+from datahub.emitter.mcp_builder import (
+    ContainerKey,
+)
+from datahub.ingestion.source.common.subtypes import MLAssetSubTypes
+from datahub.metadata.urns import (
+    CorpUserUrn,
+    GlossaryTermUrn,
+    TagUrn,
+)
+from datahub.sdk.container import Container
+from datahub.sdk.dataset import Dataset
+from datahub.sdk.main_client import DataHubClient
+from datahub.sdk.mlmodel import MLModel
+from datahub.sdk.mlmodelgroup import MLModelGroup
+
+run_id = "simple_training_run"
+run_name = "Simple Training Run"
+experiment_id = "airline_forecast_experiment"
+experiment_name = "Airline Forecast Experiment"
+model_id = "arima_model"
+model_name = "ARIMA Model"
+model_group_id = "airline_forecast_models_group"
+model_group_name = "Airline Forecast Models Group"
+
+
+def init_client():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--token", required=False, help="DataHub access token")
+    parser.add_argument(
+        "--server_url",
+        required=False,
+        default="http://localhost:8080",
+        help="DataHub server URL (defaults to http://localhost:8080)",
+    )
+    args = parser.parse_args()
+
+    client = DataHubClient(token=args.token, server=args.server_url)
+    return client
+
+
+if __name__ == "__main__":
+    # init client
+    client = init_client()
```

Comment:
let's use `DataHubClient.from_env()` - the argparse stuff just makes the example harder to read